test/rest_api/test_storage_service.py

```diff
@@ -20,6 +20,26 @@ def new_keyspace(cql, this_dc):
     cql.execute(f"CREATE KEYSPACE {name} WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 1 }}")
     return name
 
+def test_storage_service_keyspaces(cql, this_dc, rest_api):
+    keyspace = new_keyspace(cql, this_dc)
```

Comment:
We have a better mechanism since this one was implemented.
Let's use `with new_test_keyspace`, as in https://github.com/scylladb/scylla/blob/1cb64de8d8ff63b6eac3f014d46fd854414b6f6e/test/rest_api/test_storage_service.py#L103 instead, so it gets automatically dropped (also on error)